types/wlr_screencopy_v1.c

```diff
@@ -52,8 +126,26 @@ static void frame_handle_output_precommit(struct wl_listener *listener,
 		return;
 	}
 
+	struct screencopy_damage *damage = NULL;
+	if (frame->with_damage) {
+		damage = get_screencopy_damage(frame->manager, output);
```

Comment:
This creates one global `screencopy_damage` per output. This breaks when there are multiple clients capturing the same output.